### ISSUE TITLE
Open recent files

### ISSUE THREAD

**Issue by coristig** [yhat/rodeo#196]
Add a "Open Recent" to the Rodeo navbar.

**Comment by barcadad** [comment]
Yes, I would also love to have an "Open Recent" in the File navbar, which would be especially useful since files don't reopen after Rodeo restarts.
